sections/Discussion/process-reflections.tex

```diff
@@ -19,18 +19,22 @@ \section{Reflecting on the process}
 \\\\
 \subsection{Evaluation of Essence}
 One of the main things that was used from Essence was the roles.
-While Essence includes things like potential and filters for the pre-project, these were not introduced until the pre-project phase of the semester had passed, and was deemed less meaningful to use at these stages.\\
-Additionally, since not all group members took the software innovation class where Essence was presented, it took a lot of work to relay the information since the remaining group members would essentially have to learn the curriculum of another course on top of the ones they were already following.
+While Essence includes things like potential and filters for the pre-project, these were not introduced until the pre-project phase of the semester had passed, and were deemed less meaningful to use at these stages.
+Potentials and filters are about finding potential components used for the solution, and then narrow down to a few concrete potentials.\\
+Additionally, since not all group members took the software innovation course where Essence was presented, it took a lot of work to relay the information since the remaining group members would essentially have to learn the curriculum of another course on top of the ones they were already following.
 \\\\
-Our experience with the Essence roles was that it was quite similar to the roles present in the well-known Scrum approach.
+Our experience with the Essence roles was that they were quite similar to the roles present in the well-known Scrum approach.
 The challenger, in Essence, has a similar role as a product owner, the anchor as the scrum master, and the responder corresponds to being a member of the development team.
 The primary addition that Essence has to roles is the addition of the child role.
 This role was supposed to be a fluctuating role which allowed group members to come with feedback and encourage divergent thinking.\\
-However, since a lot of the pre-project parts of Essence were skipped, the child role did not make a lot of sense and quickly ended up becoming a joke to suggest silly ideas rather than a useful tool.
+However, since many of the pre-project parts of Essence were skipped, the child role did not make a lot of sense and quickly ended up becoming a joke to suggest silly ideas rather than a useful tool.
 \\\\
-Had we known about all parts of Essence before starting the project, there is a large possibility that it would have been significantly more useful, and that going through the pre-project models would have helped in increasing the quality of the final system.
+Had we known about all parts of Essence before starting the project, there is a large possibility that it would have been significantly more useful, and that going through the pre-project with models would have helped in increasing the quality of the final system.
```

Comment:
```suggestion
Had we known about all parts of Essence before starting the project, there is a large possibility that it would have been significantly more useful, and that going through the pre-project aspects would have helped in increasing the quality of the final system.
```